backend/spring-routie/src/main/java/routie/routie/controller/dto/response/RoutieReadResponse.java

```diff
@@ -24,13 +35,17 @@ public static RoutieReadResponse from(final Routie routie, final List<Route> rou
     private record RoutiePlaceResponse(
             Long id,
             int sequence,
-            Long placeId
+            Long placeId,
+            @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm") LocalDateTime arriveDateTime,
+            @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm") LocalDateTime departureDateTime
     ) {
-        private static RoutiePlaceResponse from(final RoutiePlace routiePlace) {
+        private static RoutiePlaceResponse from(final RoutiePlace routiePlace, final TimePeriod timePeriod) {
             return new RoutiePlaceResponse(
                     routiePlace.getId(),
                     routiePlace.getSequence(),
-                    routiePlace.getPlace().getId()
+                    routiePlace.getPlace().getId(),
+                    timePeriod != null ? timePeriod.startTime() : null,
+                    timePeriod != null ? timePeriod.endTime() : null
```

Comment:
개인적으로 삼항 연산자를 사용할 때는 == 이 먼저 나오는 것이 가독성이 좋았습니다.
차니 의견은 어떠신가요?
```suggestion
                    timePeriod == null ? null : timePeriod.startTime(),
                    timePeriod == null ? null : timePeriod.endTime()
```